### ISSUE TITLE
[Question] compound shortcut 

### ISSUE THREAD

**Issue by carlosmiei** [koekeishiya/skhd#218]
Hello,

I'm trying to set up some shortcuts like this:

```
alt - c - 4  : (...)
```
Isn't this a valid "shortcut"?  (alt + letter + number)If I add it, besides not working skhd starts malfunctioning too. 

Thanks guy!

**Comment by es183923** [comment]
no, its not a valid shortcut

**Comment by carlosmiei** [comment]
Hi @es183923, thanks for answering, is that a way to make it "valid"?

**Comment by es183923** [comment]
I don't think so, but you could do this with modes, but the behavior would be slightly different. Then, you would press `alt-c`, then press `4`. The config would look like this:

```sh
:: your_mode # or :: your_mode : script
alt - c ; your_mode
your_mode < 4 : another_script
```
`your_mode` can be whatever you want it to be, `script` gets triggered when you enter `your_mode` by doing `alt - c`, and `another-script` gets triggered when you press `4` in `your_mode`.

**Comment by carlosmiei** [comment]
@es183923 actually, that might work, thank you!
